scripts/release/promote/wheel_to_pypi.sh

```diff
@@ -34,13 +41,25 @@ fi
 
 for pkg in ${pkgs_to_promote}; do
     pkg_basename="$(basename "${pkg//linux/manylinux1}")"
+    orig_pkg="${tmp_dir}/${pkg_basename}"
     (
         set -x
         # Download package, sub out linux for manylinux1
-        curl -fsSL -o "${pkg_basename}" "https://download.pytorch.org/whl/${pkg}"
+        curl -fsSL -o "${orig_pkg}" "https://download.pytorch.org/whl/${pkg}"
+    )
+
+    if [[ -n "${VERSION_SUFFIX}" ]]; then
+        OUTPUT_DIR=${output_tmp_dir} ${DIR}/prep_binary_for_pypi.sh "${orig_pkg}"
```

Comment:
do we need a double quote here?
```suggestion
        OUTPUT_DIR="${output_tmp_dir}" ${DIR}/prep_binary_for_pypi.sh "${orig_pkg}"
```